examples/2-advanced/stage_4_galaxies.ipynb

```diff
@@ -40,6 +43,9 @@
     "import glass\n",
     "import glass.ext.camb\n",
     "\n",
+    "# set up matplotlib to produce consistent figures\n",
+    "mpl.rc_file(pathlib.Path.cwd().parents[1] / \".matplotlibrc\")\n",
+    "\n",
```

Comment:
I though it automatically picks up the config from the config file in the current working directory? Is something like that not possible?

Reply:
From what I've read it has to be `maplotlibrc` (i.e. without the dot). Plus we have notebooks in two different subfolders, so I think it's safer to force load.